client/state/sharing/services/test/selectors.js

```diff
@@ -38,11 +38,11 @@ describe( 'selectors', () => {
 						jetpack_support: true,
 						type: 'publicize',
 					},
-					eventbrite: {
-						ID: 'eventbrite',
+					googlePhotos: {
```

Comment:
Probably could have just left the tests as eventbrite, as only mocking anyway, but  changed just to avoid any confusion